docs/user/faq.rst

```diff
@@ -25,3 +25,36 @@ interface. This DNS server must be announced in router advertisements (using
 on *batman-adv*. If your mesh does not have global IPv6 connectivity, you can setup
 your *radvd* not to announce a default route by setting the *default lifetime* to 0;
 in this case, the *radvd* is only used to announce the DNS server.
+
+.. _faq-lost-settings:
+
+Which settings are retained or migrated upon an update?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Gluon provides a :doc:`../dev/web/config-mode`, which allows the configuration via webinterface.
+Under the hood it uses UCI and sets options according to the users input.
+
+There's a recurring misconception about all UCI settings being retained and migrated across
+updates unconditionally. A settings presence does not imply this on its own.
+
+Only if a setting is either configurable via the setupmode, or explicitly referenced
+in this documentation - the wiki or other inofficial places do not count - it is supposed to
+survive the update process.
+
+All other options might disappear or lose functionality in any update.
```

Comment:
```suggestion
All other options may be overridden, might disappear or lose functionality after a firmware update or already
after calling gluon-reconfigure.
```

Reply:
resolved in b004276601ba242d1a338c9ca182f8feaccda2d2